packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -31,15 +32,33 @@ class NodeMessageBase(BaseModel):
 class LoggerRabbitMessage(RabbitMessageBase, NodeMessageBase):
     channel_name: Literal["simcore.services.logs"] = "simcore.services.logs"
     messages: list[str]
+    log_level: int = logging.INFO
 
 
 class EventRabbitMessage(RabbitMessageBase, NodeMessageBase):
     channel_name: Literal["simcore.services.events"] = "simcore.services.events"
     action: RabbitEventMessageType
 
 
+class ProgressType(Enum):
+    COMPUTATION_RUNNING = auto()  # NOTE: this is the original only progress report
+
+    CLUSTER_UP_SCALING = auto()
+    SIDECARS_PULLING = auto()
+    SERVICE_INPUTS_PULLING = auto()
+    SERVICE_OUTPUTS_PULLING = auto()
```

Comment:
This if fine when the service is starting or closing.

There is also the case where the sidecar detects changes in the outputs ports uploads ports in batches of various length. I think that might require a bit more work to be more informative. Let's see how this works now.

Reply:
@GitHK : I forgot to mention that some parts are not implemented such output pushing. I will need your insights on how to add this.